### DISCUSSION TITLE
Add an “always visible” checkbox for gameplay elements in skin editor

### DISCUSSION THREAD

**Discussion by McMetal69** [ppy/osu#24193] [Ideas]
I’ve been enjoying osu lazer lately however, I wish it would be possible to show certain gameplay elements while hiding others.

For instance, I want the unstable bar and the progress-piechart to be visible at all times while playing. While still being able to check acc/other skin elements during breaks.

SOLUTION: 
Add an “always visible” checkbox for the selected gameplay element that when ticked makes the element ignore the  hud overlay visibility options and instead always stays visible.

With this it would be possible for the player to choose every gameplay element they wish to  permanently have onscreen while keeping the less important/distracting elements (score, health, pictures and such) easily toggle-able Ingame by simply choosing between always visible/ only during breaks / never visible 
just like in current Lazer.

Only way to have a permanent ur bar/piechart as of now is to remove all the other gameplay elements and turn on always visible as far as I know.

**Comment by Theighlin** [comment]
https://github.com/ppy/osu/issues/12039#issuecomment-856453309
